README.md

```diff
@@ -1,18 +1,23 @@
+# Todo
```

Comment:
@rindrajosia also mentioned updating your README file.

Checkout out the below link for help, feel free to follow this [Microverse template](https://github.com/microverseinc/readme-template) from pathwright

Heads-up: Add screenshot of your app to the README file